File: data/src/main/java/com/ykb/app/cryptotrader/data/dao/ParameterDao.java
package com.ykb.app.cryptotrader.data.dao;

import com.ykb.app.cryptotrader.data.model.Parameter;
import com.ykb.app.cryptotrader.data.repo.ParameterRepo;
import org.springframework.stereotype.Component;

@Component
public final class ParameterDao extends Dao<Parameter, String> {

    public ParameterDao(ParameterRepo repo) {
        super(repo);
    }

}

File: library/data/data-core/src/main/kotlin/com/ykb/app/cryptotrader/data/core/dao/ParameterDao.kt
package com.ykb.app.cryptotrader.data.core.dao

import com.ykb.app.cryptotrader.data.base.Dao
import com.ykb.app.cryptotrader.data.core.model.Parameter
import com.ykb.app.cryptotrader.data.core.repo.StrategyRepo
import org.springframework.stereotype.Component

@Component
class ParameterDao(repo: StrategyRepo) : Dao<Parameter, String>(repo)